### PR TITLE
FEAT: medium and high levels screen add

### PR DESCRIPTION
**Pull request by samyakshah3008**

### CODE REVIEW

File: quizrulemedium/quizrulemedium.html [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+            <div class="rule">You get 30 seconds to answer each question.</div>
+            <div class="rule">Score 70% or above to unlock the next level</div>
+           <a href="/styles/questions/question1.html"><button class="start-quiz-btn margin-top">Start Quiz</button></a>
+        </div>
+    </main>
+
+
+</body>
+
+</html>
```
Comment (pritam-kr) [inline quizrulemedium/quizrulemedium.html:36]: break line after last line of code

File: style.css [hunk 2]
```diff
@@ -90,3 +90,11 @@
     color: var(--success-color);
     font-weight: bold;
```
Comment (pritam-kr) [inline style.css:91]: you can use variables for this font-weight but its ok

File: medium-level.html [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+                        you know about IPL</div>
+
+                    <a href="/quizrulemedium/quizrulemedium.html"><button class="playnow-btn margin-top">Play
+                            now</button></a>
+
+                </div>
+            </div>
+
+            <div class="card">
+                <img src="./styles/rcb-logo.png" alt="icc-worldcup-trophy" class="card-img">
```
Comment (pritam-kr) [inline medium-level.html:59]: it is good practice to use alt attributes.

**Review by pritam-kr — APPROVED** [review]
Good to Merge Samyak 
I gave some little suggestions. All looks good to me. UI is also clean. You can merge it

File: medium-level.html [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+
+                </div>
+            </div>
+        </div>
+
+
+    </main>
+</body>
+
+</html>
```
Comment (varnita619) [inline medium-level.html:78]: You can install addnewline extension to add a new line at the end of your file

**Review by varnita619 — APPROVED** [review]

**Pull request merged**